fail2ban/tests/sockettestcase.py

```diff
@@ -83,3 +90,17 @@ def testSocketForce(self):
 		self.server.stop()
 		serverThread.join(1)
 		self.assertFalse(os.path.exists(self.sock_name))
+
+
+class ClientMisc(unittest.TestCase):
+
+	def testPrintFormattedAndWiki(self):
+		# redirect stdout to devnull
+		saved_stdout = sys.stdout
+		sys.stdout = open(os.devnull, 'w')
+		try:
+			protocol.printFormatted()
+			protocol.printWiki()
+		finally:
+			# restore stdout
+			sys.stdout = saved_stdout
```

Comment:
here you just do smoke testing, no printouts checked anyhow etc.  with a context manager you can assess either what was printed is actually correct etc.  So far for testing logging we had custom class with fixtures to grab logged output etc, context manager allows in concise manner to surround specific piece of code.